packages/core/src/DdSdkReactNativeConfiguration.tsx

```diff
@@ -250,6 +252,18 @@ export class DdSdkReactNativeConfiguration {
      */
     public trackBackgroundEvents: boolean = DEFAULTS.trackBackgroundEvents;
 
+    /**
+     * Enables RUM for logs.
+     * By default, RUM is enabled for logs.
+     */
+    public enableRumForLogs: boolean = DEFAULTS.enableRumForLogs;
+
+    /**
+     * Enables Traces for logs.
+     * By default, Traces is enabled for logs.
+     */
+    public enableTracesForLogs: boolean = DEFAULTS.enableTracesForLogs;
```

Comment:
we need to align API naming with native SDKs, these properties are called differently there.